nautobot/extras/jobs.py

```diff
@@ -483,6 +484,15 @@ def as_form(cls, data=None, files=None, initial=None, approval_view=False):
             dryrun_default = cls.dryrun_default
             task_queues = cls.task_queues
 
+        # Initialize job_queue choices
+        form.fields["job_queue"] = DynamicModelChoiceField(
+            display_field="display_with_worker_count",
```

Comment:
Ah, I see.

Given that we expect a relatively small number of distinct queues, it *might* be OK to do a static ModelChoiceField instead of a DynamicModelChoiceField so that we don't need to "pollute" the REST API with this attribute?

Alternately, maybe the `display_with_worker_count` should just become the baseline `display` for the JobQueue model?